### PR TITLE
Table: To/From Postgres

### PR DESCRIPTION
**Pull request by jburchard**
Add the following convenience methods:

`Table.to_postgres()`
`Table.from_postgres()`

### CODE REVIEW

File: parsons/etl/tofrom.py [hunk 1]
```diff
@@ -419,10 +419,38 @@ def to_redshift(self, table_name, username=None, password=None, host=None,
...
+        rs = Redshift(username=username, password=password, host=host, db=db, port=port)
         rs.copy(self, table_name, **copy_args)
 
+    def to_postgres(self, table_name, username=None, password=None, host=None,
+                    db=None, port=None, **copy_args):
+        """
+        Write a table to a Postgres database.
+
+        Args:
+            username: str
```
Comment (eliotst) [inline parsons/etl/tofrom.py]: Missing documentation for the `table_name`.

File: parsons/etl/tofrom.py [hunk 2]
```diff
@@ -596,7 +624,31 @@ def from_redshift(cls, query, username=None, password=None, host=None,
 
         from parsons.databases.redshift import Redshift
         rs = Redshift(username=username, password=password, host=host, db=db, port=port)
-        return rs.query(query)
+        return rs.query(sql)
+
+    @classmethod
+    def from_postgres(cls, sql, username=None, password=None, host=None, db=None, port=None):
```
Comment (eliotst) [inline parsons/etl/tofrom.py]: Missing the `pg_pass` argument.

File: parsons/etl/tofrom.py [hunk 3]
```diff
@@ -419,10 +419,38 @@ def to_redshift(self, table_name, username=None, password=None, host=None,
         """  # noqa: W605
 
         from parsons.databases.redshift import Redshift
-        rs = Redshift(
-            username=username, password=password, host=host, db=db, port=port)
+        rs = Redshift(username=username, password=password, host=host, db=db, port=port)
         rs.copy(self, table_name, **copy_args)
 
+    def to_postgres(self, table_name, username=None, password=None, host=None,
+                    db=None, port=None, **copy_args):
```
Comment (eliotst) [inline parsons/etl/tofrom.py]: Missing the `pg_pass` argument documented below.

**Comment by jburchard** [comment]
I'd like to spend a little bit more time with the `pg_pass` broadly as part of the Postgres class, so I'm removing for the moment.

**Pull request merged**